draft-ietf-mls-architecture.md

```diff
@@ -272,13 +272,44 @@ all users, for all group sizes, including groups of only two clients.
 
 # General Setting
 
+## Terminology
```

Comment:
Perhaps rename this to "Protocol Overview"?

Reply:
Sure